packages/modules/providers/payment-stripe/package.json

```diff
@@ -25,17 +25,17 @@
   "devDependencies": {
     "@medusajs/medusa": "^1.20.3",
     "@types/stripe": "^8.0.417",
-    "awilix": "^8.0.1",
     "cross-env": "^5.2.1",
     "jest": "^29.7.0",
     "rimraf": "^5.0.1",
     "typescript": "^5.3.3"
   },
   "peerDependencies": {
-    "@medusajs/medusa": "^1.12.0"
+    "@medusajs/medusa": "^1.12.0",
```

Comment:
question: Can we remove this one as well?

Reply:
Good catch. we don't need that anymore in plugins/providers.